src/oci/image/config.rs

```diff
@@ -71,13 +71,13 @@ pub struct Config {
     history: Vec<History>,
 }
 
-pub fn parse_config(content: &str) -> Result<Config, Box<dyn error::Error>> {
+pub fn from_str(content: &str) -> Result<Config, Box<dyn error::Error>> {
```

Comment:
Why aren't you implementing FromStr?

https://doc.rust-lang.org/std/str/trait.FromStr.html

Reply:
In fact, it's already there from the derivation, getting rid of this useless method.

Reply:
Good, that's what I thought.